app/src/main/java/de/moyapro/idle/domain/Species.kt

```diff
@@ -48,7 +52,7 @@ class Species(
 
 fun defaultSpecies(name: String = "DefaultSpecies"): Species {
     return Species(
-        name, mutableListOf(
+        name, Feature(
```

Comment:
Das passt. Aber wird es in Zukunft vorab konfigurierte (benannte) Feature geben, ähnlich den Traits?

Reply:
wird also noch ausgewürfelt... ;)